Minimatch.Portable.Net40/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,5 @@
+﻿using System.Reflection;
+
+[assembly: AssemblyTitle("Minimatch")]
```

Comment:
It was just a habit of mine, because I usually have more than one assembly in the solution for different things and therefore need multiple product names. Here it's not really needed (yet), but I wanted to keep the AssemblyInfo.cs with some contents.